client/uploadScrambles.js

```diff
@@ -105,12 +105,11 @@ function extractJsonFromZip(filename, zipId, pw, cb) {
         } else {
           cb("Wrong password");
         }
-        return;
-      } else {
-        throw error;
       }
+      console.error("Meteor.call() error: " + err);
```

Comment:
Yes, I didn't see that the code made that distinction.

Though when I see it now, I actually prefer logging all errors. Makes the code simpler and is consistent.